deploy/platform/kubernetes/features.mk

```diff
@@ -81,3 +81,27 @@ undeploy.feature-kubernetes-monitor:
 	@kubectl delete --ignore-not-found -f https://raw.githubusercontent.com/kubernetes/kube-state-metrics/v2.2.4/examples/standard/cluster-role-binding.yaml
 	@kubectl delete --ignore-not-found -f https://raw.githubusercontent.com/kubernetes/kube-state-metrics/v2.2.4/examples/standard/service.yaml
 	@kubectl delete --ignore-not-found -f https://raw.githubusercontent.com/kubernetes/kube-state-metrics/v2.2.4/examples/standard/deployment.yaml
+
+# @feature: java agent injector
```

Comment:
Can you use this comment pattern: `# @feature: <feature-name>; <what and why we need this step in this feature>`. The same for other comments added in this PR.

Reply:
OK, updated it.